bin/asaph_localize

```diff
@@ -134,19 +135,21 @@ def manhattan_plot(plot_fl, snp_pvalues, boundaries=None, y_limit=None):
 
     fig = plt.gcf()
 
-    df_insign = df[df["is_significant"] == 0]
+   df_insign = df[df["is_significant"] == 0]
```

Comment:
I think this line should align with the if statement below it.  Can you double check that please?